README.md

```diff
@@ -72,36 +72,58 @@ solver.attributes.linear_solver = "ma27"
 
 ## Quick start
 
-### Generating random instances
+### Building and solving an OPF problem
 
 ```julia
-using Random, PGLib, PowerModels
+using PGLib, PowerModels
 using OPFGenerator
-PowerModels.silence()
+using Ipopt
+
+# Load data from PGLib case
+pm_network = PowerModels.make_basic_network(pglib("14_ieee"))
+data = OPFGenerator.OPFData(pm_network)
+
+# Build OPF optimization model
+# To switch formulations, replace ACOPF with, e.g., DCOPF or SOCOPF
+opf = OPFGenerator.build_opf(OPFGenerator.ACOPF, data, Ipopt.Optimizer)
+
+# Solve and extract result
+OPFGenerator.solve!(opf)
+res = OPFGenerator.extract_result(opf)
+```
 
-rng = MersenneTwister(42)
 
-old_data = make_basic_network(pglib("3_lmbd"))
+### Generating random OPF instances
+
+```julia
+using Random 
+using PGLib, PowerModels
+using OPFGenerator
+
+# Load data from PGLib case
+pm_network = PowerModels.make_basic_network(pglib("14_ieee"))
+data = OPFGenerator.OPFData(pm_network)
 
 # Load scaler using global scaling + uncorrelated LogNormal noise
 config = Dict(
     "load" => Dict(
-        "noise_type" => "ScaledLogNormal",
+        "noise_type" => "ScaledUniform",
         "l" => 0.8,
         "u" => 1.2,
-        "sigma" => 0.05,        
+        "sigma" => 0.10,       
     )
 )
-opf_sampler  = SimpleOPFSampler(old_data, config)
+opf_sampler = SimpleOPFSampler(data, config)
 
 # Generate a new instance
+rng = MersenneTwister(42)
 new_data = rand(rng, opf_sampler)
 
-old_data["load"]["1"]["pd"]  # old 
-1.1
+data.pd[1] # old 
+0.217
 
-new_data["load"]["1"]["pd"]  # new
-1.1596456429775048
+new_data.pd[1]  # new
+0.21480423013573954
```

Comment:
I got `0.2178586684315607` with 1.10.7. May be worth putting a disclaimer that this value varies per Julia version

Reply:
Fair point